packages/browsers/src/CLI.ts

```diff
@@ -396,6 +400,45 @@ export class CLI {
           );
         },
       )
+      .command(
+        'list',
+        'List all installed browsers in the cache directory',
+        yargs => {
+          this.#definePathParameter(yargs);
+          yargs.example(
+            '$0 list',
+            'List all installed browsers in the cache directory',
+          );
+          if (this.#allowCachePathOverride) {
+            yargs.example(
+              '$0 list --path /tmp/my-browser-cache',
+              'List browsers installed in the specified cache directory',
+            );
+          }
+        },
+        async argv => {
+          const args = argv as unknown as ListArgs;
+          const cacheDir = args.path ?? this.#cachePath;
+
+          try {
+            const cache = new Cache(cacheDir);
+            const browsers = cache.getInstalledBrowsers();
+
+            if (browsers.length === 0) {
+              console.log('No browsers installed.');
+              return;
+            }
+
+            for (const browser of browsers) {
+              console.log(
+                `${browser.browser}@${browser.buildId} (${browser.platform}) ${browser.executablePath}`,
+              );
+            }
+          } catch (error) {
+            console.error(`Failed to list browsers: ${error}`);
```

Comment:
I think we probably do not want to catch an error here so that the whole command exits with an error status code.